### DISCUSSION TITLE
Suspended trading Stocks

### DISCUSSION THREAD

**Discussion by cotton1234** [BennyThadikaran/eod2#98] [Ideas]
Hello Benny,

Today while working on the setup, i came across a stock rnaval for which trading was suspended in 2023 but it is there in daily eod2_data folder. Is it possible to remove these stocks from the folder.

Thanks
Saurabh

**Comment by BennyThadikaran** [comment]
Usually if a stock has no trading data for more than 365 days, it is removed. I keep the data incase of temporary suspensions or awaiting some corporate actions.

I have not found any info on NSE website about recently suspended stocks. I could use such information to clean eod2_data.

**Reply by cotton1234** [reply]
Fine Benny, will handle the same in my code.
